docs/docusaurus/docs/reference/expectations/result_format.md

```diff
@@ -235,21 +225,19 @@ For example:
 ```python name="tests/integration/docusaurus/reference/core_concepts/result_format/result_format_summary_example_agg"
 ```
 
-Will return the following output:
+Returns the following output:
 
 ```python name="tests/integration/docusaurus/reference/core_concepts/result_format/result_format_summary_example_agg_output"
 ```
 
 
 ### Behavior for `COMPLETE`
-A `result` is generated with all available justification for why an Expectation was met or not. The format is  
-intended for debugging pipelines or developing detailed regression tests.
 
-Great Expectations has standard behavior for support for describing the results of `column_map_expectation` and
-`ColumnAggregateExpectation` Expectations.
+A `result` is generated with all available justification for why an Expectation was successful or unsuccessful. The format is intended for debugging pipelines or developing detailed regression tests.
+
+Great Expectations has standard behavior for support for describing the results of `column_map_expectation` and `ColumnAggregateExpectation` Expectations.
```

Comment:
This reads a bit awkwardly with the double 'for' in it.  Maybe something like:
```suggestion
Great Expectations implements standard behaviors to support describing the results of `column_map_expectation` and `ColumnAggregateExpectation` Expectations.
```
...not a strong recommendation.  Just seemed the wording here could be better.